test/helpers/kubectl.go

```diff
@@ -136,6 +139,15 @@ var (
 		"nodeinit.enabled":       "true",
 	}
 
+	eksENIHelmOverrides = map[string]string{
+		"eni":                        "true",
+		"ipam.mode":                  "eni",
+		"egressMasqueradeInterfaces": "eth0",
+		"tunnel":                     "disabled",
+		"nodeinit.enabled":           "true",
+		"ipv6.enabled":               "false",
```

Comment:
Had to disable this because it's included in the `defaultHelmOptions` and not supported in ENI mode. The rest of the values are taken from the official Cilium docs.